RELEASES.md

```diff
@@ -8,6 +8,9 @@ Unreleased.
 
 ### Changed
 
+The `-S common` flag is renamed to `-S cli`, to better reflect that it provides
+the wasi-cli APIs. `-S common` is still accepted, with a warning.
```

Comment:
Mind updating this following the same format as below with bullets and a link to the PR?